packages/qwik/src/types/form.ts

```diff
@@ -100,3 +100,8 @@ export type FormStore<
   invalid: boolean;
   response: Response;
 };
+
+export type FormValues<TFormState extends FormStore<any, any, any>> =
```

Comment:
Please rename `TFormState` to `TFormStore` and add a comment above it describing what the type does. For example:

```ts
/**
 * Extracts the field values from the form store.
 */
```

Reply:
Done, pretty much same comment as in Solid, only changing state for store